plugins/search/providers/elastic_search/lib/model/items/ESearchBaseOperator.php

```diff
@@ -55,7 +55,7 @@ public function setSearchItems($searchItems)
 
 	public static function createSearchQuery($eSearchItemsArr, $boolOperator, &$queryAttributes, $eSearchOperatorType = null)
 	{
-		if (!$eSearchItemsArr || !count($eSearchItemsArr))
+		if (empty($eSearchItemsArr))
```

Comment:
why changing this?

Reply:
Initially when building the new nested query object for accuracy and language, the new structure broke this condition and the best way to overcome this was to change it to empty() which is functionally equivalent. I later did more changes to how i create this nested object and didnt change it back since it is equivalent. I will revert this change and continue testing to make sure nothing breaks